app/src/lib/dispatcher/dispatcher.ts

```diff
@@ -170,8 +170,14 @@ export class Dispatcher {
 
     const repositoryIDs = localRepositories.map(r => r.id)
     await this.dispatchToSharedProcess<ReadonlyArray<number>>({ name: 'remove-repositories', repositoryIDs })
-
-    this.showFoldout({ type: FoldoutType.Repository })
+    await this.loadRepositories().then(response => {
```

Comment:
I tried returning `return (await repositoriesStore.getRepositories()).length` in `app/src/shared-process/index.ts`in line 87 to no such luck. I come from an Angular/.Net background I am new to Electron and React. Am I missing something. I that the wrong approach?